### PR TITLE
Transfer assets to Homebridge

### PR DESCRIPTION
**Pull request by NorthernMan54**
## :recycle: Current situation

*Describe the current situation. Explain current problems, if there are any. Be as descriptive as possible (e.g., including examples or code snippets).*

## :bulb: Proposed solution

*Describe the proposed solution and changes. How does it affect the project? How does it affect the internal structure (e.g., refactorings)?*

## :gear: Release Notes

*Provide a summary of the changes or features from a user's point of view. If there are breaking changes, provide migration guides using code examples of the affected features.*

## :heavy_plus_sign: Additional Information
*If applicable, provide additional context in this section.*

### Testing

*Which tests were added? Which existing tests were adapted/changed? Which situations are covered, and what edge cases are missing?*

### Reviewer Nudging

*Where should the reviewer start? what is a good entry point?*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ Join the [Official Homebridge Discord](https://discord.gg/Cmq8a44) community and
 
 ## License
 
-Copyright (C) 2017-2022 oznu
+Copyright (C) 2017-2023 homebridge
```
Comment (oznu) [inline README.md:115]: Add a new line above while retaining the existing ones 😀

**Review by Supereg — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (NorthernMan54) [reply README.md:115]: Like this ?

```
Copyright (C) 2023 homebridge
Copyright (C) 2017-2022 oznu
```

**Comment by NorthernMan54** [comment]
Also I left the README container repository's ( documentation ) alone until we had publishing 100% transferred.  Did not want to leave people without ability to use the package.

File: README.md (hunk 1 above)
Comment (oznu) [reply README.md:115]: LGTM
